test/expected_loglik.jl

```diff
@@ -37,4 +37,12 @@
         results = map(m -> ApproximateGPs.expected_loglik(m, y, q_f, lik), methods)
         @test all(x -> isapprox(x, results[end]; atol=1e-6, rtol=1e-3), results)
     end
+
+    @test ApproximateGPs.expected_loglik(
+        MonteCarlo(), zeros(10), q_f, GaussianLikelihood()
+    ) isa Real
+    @test ApproximateGPs.expected_loglik(
```

Comment:
Could you make this test check that this result is close to the exact version?

Reply:
That's already checked in the tests above

Reply:
Oh, right, yes. But then why does it also need to be tested here? Which line is it covering that wasn't previously covered?

Reply:
It's for the default constructors `MonteCarlo()` and `GaussHermite()`, which don't _really_ need to be tested separately but I suppose there's no reason not to

Reply:
Ohhhhhhhhhhh I see. Yeah, could to ensure that they're covered. I'm happy with these tests.